### PR TITLE
Python: Add iFrame based Task Module sample

### LINKED ISSUE #2574: Add iFrame based Task Module samples for remaining languages
On the back of the .NET sample https://github.com/microsoft/BotBuilder-Samples/pull/2546 - add this sample to the other languages.

- [ ] Python (https://github.com/microsoft/BotBuilder-Samples/tree/trboehre/teamstasks)
- [ ] JS
- [x] Java

### PR DESCRIPTION
**Pull request by tracyboehrer**
Fixes #2574

### CODE REVIEW

**Comment by tracyboehrer** [comment]
@EricDahlvang @axelsrz There is still the problem Eric has when running it.  It runs as expected for me and Axel though.

File: samples/python/54.teams-task-module/bots/teams_task_module_bot.py [hunk 1]
```diff
@@ -2,84 +2,162 @@
...
+    @staticmethod
+    def __set_task_info(task_info: TaskModuleTaskInfo, ui_constants: UISettings):
+        task_info.height = ui_constants.height
+        task_info.width = ui_constants.width
+        task_info.title = ui_constants.title
+
+    @staticmethod
+    def __get_task_module_hero_card_options() -> Attachment:
+        buttons = [
+            CardAction(
```
Comment (EricDahlvang) [inline samples/python/54.teams-task-module/bots/teams_task_module_bot.py:112]: When I ran this, I had to json.dumps the value:

```python
CardAction(
                type="invoke",
                title=card_type.button_title,
                value=json.dumps({"type": "task/fetch", "data": card_type.id}),
            )
```

I have no idea why... does it break for your guys if you make this change?

Comment (tracyboehrer) [reply samples/python/54.teams-task-module/bots/teams_task_module_bot.py:112]: No

Comment (tracyboehrer) [reply samples/python/54.teams-task-module/bots/teams_task_module_bot.py:112]: Sorry.  Never tried that.  Shouldn't be necessary.  We can't explain why of the 3 of us, you have problems without it.

Comment (tracyboehrer) [reply samples/python/54.teams-task-module/bots/teams_task_module_bot.py:112]: Seems to work with it though.  Change pushed.

**Pull request merged**